### ISSUE TITLE
Brain dump on output data

### ISSUE THREAD

**Issue by aforren1** [aforren1/finger-5#2]
I'm struggling with the _most_ right way to save data. Some hierarchical format (HDF5, mat file?) seems nice for keeping things organized, but makes reading later less trivial (or I don't know what it'll look like).

*\* Note: all relative times are relative to the onset of the trial (was thinking relative to the sound onset, but that doesn't sit as well)

```
 id/ (numeric)
     day/ (numeric)
         block/ (numeric)
             block type (string)
             swap (bool)
             image type (string)
             tfile header (string array)
             tfile (matrix)
             trial/ 
                 any presses (on/off relative/absolute time, choice)
                 image presentation (index, on relative/absolute time)
                 sound onset time (relative, absolute)
```

HDF5 would be fancy, but it looks like the MATLAB interface to the file format is different from Octave. Cell arrays are a common denominator, but a little TLC needs to be taken. In MATLAB, use `save(filename, variables, '-v7')`, and in Octave, use `save('-mat7-binary', filename, variables)`.

**Comment by aforren1** [comment]
Basic idea:

```
presses(1:5) = struct('index', int16(-1), 'rel_time_on', -1,...
                 'rel_time_off', -1, 'abs_time_on', -1, ...
                 'abs_time_off', -1);
images(1) = struct('index', int16(-1), 'rel_time_on', -1, ...
                'abs_time_on', -1);
sounds(1) = struct('rel_time_on', -1, ...
                'abs_time_on', -1);

trial(1:150) = struct('presses', presses, ...
               'images', images, ...
               'sounds', sounds, ...
               'forces', zeros(600, 11));

block(1:10) = struct('trial', trial, ...
               'block_type', '', ...
               'swapped', false,...
               'image_type', '',...
               'tfile_header', [], ...
               'tfile', []);

day(1:10) = struct('block', block);

id = struct('day', day);
```

However, this little puppy is ~ 800 MB, which makes it a less-than-ideal candidate for reading in and out. Just one block, however, is roughly 8 MB (easy peasy).

**Comment by aforren1** [comment]
More or less addressed by 0ef91ef5b8bffe0433c3c530f1ea21a7e9531475
